Microsoft.Toolkit.Uwp.UI.Controls.Core/Strings/en-US/Resources.resw

```diff
@@ -193,18 +193,6 @@
     <value>Warning</value>
     <comment>Label for TextToolbar Control Warning message.</comment>
   </data>
-  <data name="WCT_BladeView_ExpandButton_Collapsed" xml:space="preserve">
```

Comment:
Forgot to update the strings, see https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3727/commits/f3547fe273a299cdfa8d2bb30ee8017817ca2bb7, not sure if you want to cherry-pick or just have that forward-fixed?